lib/membership/events.js

```diff
@@ -20,13 +20,24 @@
 'use strict';
 
 function LocalMemberLeaveEvent(member, oldStatus) {
-    this.name = LocalMemberLeaveEvent.Name;
+    this.name = this.constructor.name;
     this.member = member;
     this.oldStatus = oldStatus;
 }
 
-LocalMemberLeaveEvent.Name = 'localMemberLeave';
+function ReusableEvent(member, oldDampScore) {
```

Comment:
What is reusable here? I don't understand the connection between reusable and an old damp score, and there's nothing to explain it.

Also, should all Damp events be prefixed with Damp[ing] or sth similar?